spec/controllers/tags_controller_spec.rb

```diff
@@ -10,8 +10,10 @@
 
   describe '#index' do
     it 'returns correct JSON data' do
-      tag = create(:tag)
+      post :create, params: { tag: { name: 'tag', description: 'tag description' },
```

Comment:
Each controller test should only make one request, so this isn't a good change. I'm guessing the test was failing because the tag that was created wasn't associated to the `assignment`. However, the correct fix for this is to change the *factory* that's creating the test, under the `factories/` folder. You can read about associations with `FactoryBot` (the library we're using to create factories) at https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations